examples/braket_features/Noise_models/Noise_models_on_Amazon_Braket.ipynb

```diff
@@ -0,0 +1,908 @@
+{
```

Comment:
<p>Write out Quantum Processing Unit (QPU) the first time you use it in the notebook.</p><p> </p><p>The link for "Running quantum circuits on QPU device<a href="https://github.com/aws/amazon-braket-examples/blob/main/examples/getting_started/2_Running_quantum_circuits_on_QPU_devices.ipynb" target="_blank">s</a>" is broken. It should be https://github.com/aws/amazon-braket-examples/blob/main/examples/getting_started/2_Running_quantum_circuits_on_QPU_devices/2_Running_quantum_circuits_on_QPU_devices.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>